diff_drive_controller/doc/userdoc.rst

```diff
@@ -24,3 +24,135 @@ Other features
     Odometry publishing
     Task-space velocity, acceleration and jerk limits
     Automatic stop after command time-out
+
+
+ros2_control Interfaces
```

Comment:
I think people are mostly going to be interested in the ROS2 interfaces and parameters, the ros2_control interfaces should have a much smaller audience so I'm proposing to shift this section below the others